README.md

```diff
@@ -72,3 +72,46 @@ $ manifold run ./bin/server
 ## License
 
 Manifold's manifold-cli is released under the [BSD 3-Clause License](./LICENSE.md).
+
+## Configuration Files
+
+Manifold CLI has two files that help you codify some of your manifold settings and stacks, `.manifold.yml` for context configuration and `stack.yml` for stack configuration.  Neither are required, but can augment / simplify your use of Manifold depending on your use cases.  
+
+### .manifold.yml
+
+This hidden file can be created by a `manifold init` and keeps track of context, specifically:
+- `project`
+- `team` 
+
+By doing so  you no longer have to pass this in for an arg, nor worry about switching context via the `manifold switch` command.  It can be overwritten via `-t` for team and `-p` for project however.  
+
+This file can be checked into your source control as it contains no secrets, but keep in mind for open source projects it won't be useful to others as only you have access  to your team and project.
+
+### stack.yml
+
+This file can initialized via `manifold stack init` and keeps track of your stack, i.e what services your code base requires to exist in order to function. Example being a db, logger, etc.  You can manually craft this file, or you can use any of the following commands to create, modify and apply the stack file:
+
+- `manifold stack init` - Prompts for a project name and create an empty stack.yml file
+    - `(-p PROJECT)` set the project name
+    - `(-g)` Uses the current context to interactively iterate over the already existing services and prompt if the user would like them added to the `stack.yml` file
+      - context can be overwritten via `(-t TEAM)`
+- `manifold stack add` - add services to the yaml file, allows for specifying a service by name, or select from a list.  Acts similar to the `manifold create` command, however the `add` command does no actual provisioning, instead its results in an update to your `stack.yml` file
```

Comment:
Let's put the args here:
`manifold stack add OPTIONAL_RESOURCE_NAME`
( -product PRODUCT_NAME ) 
( -plan PLAN_NAME )
( -region REGION_NAME )  
( -title RESOURCE_TITLE )